src/definition.rs

```diff
@@ -200,6 +203,7 @@ mod tests {
             name = "Test template"
             description = "A description"
             kickstart_version = 1
+            symlinks = false
```

Comment:
If we use an option, I'd rather have it `follow_symlinks`

Reply:
much more descriptive, agreed. I'll make that change.